common/file/Util.cpp

```diff
@@ -151,8 +152,10 @@ bool FindMatchingFiles(const string &directory,
         files->push_back(str.str());
       }
     }
-    if (readdir_r(dp, &dir_ent, &dir_ent_p)) {
-      OLA_WARN << "readdir_r(" << directory << "): " << strerror(errno);
+    errno = 0;
```

Comment:
Can we just have a comment here then, explaining why we're setting it to zero (so no-one tidies it away in future).